src/GovUk.Education.ExploreEducationStatistics.Admin.Tests/Services/PublicationServiceTests.cs

```diff
@@ -32,12 +34,624 @@ namespace GovUk.Education.ExploreEducationStatistics.Admin.Tests.Services
+                await contentDbContext.SaveChangesAsync();
+            }
+
+            await using (var contentDbContext = InMemoryApplicationDbContext(contentDbContextId))
+            {
+                var publicationService = BuildPublicationService(
+                    context: contentDbContext,
+                    userService: userService.Object);
+
+                var result = await publicationService
+                    .ListPublications(permissions: true, topic.Id);
+
+                var publicationViewModelList = result.AssertRight();
+
+                var publicationViewModel = Assert.Single(publicationViewModelList);
+
+                Assert.Equal(publication.Id, publicationViewModel.Id);
+
+                Assert.False(publicationViewModel.IsSuperseded);
+
+                Assert.NotNull(publicationViewModel.Permissions);
+                Assert.True(publicationViewModel.Permissions!.CanUpdatePublication);
+                Assert.True(publicationViewModel.Permissions.CanUpdatePublicationTitle);
+                Assert.True(publicationViewModel.Permissions.CanUpdatePublicationSupersededBy);
+                Assert.True(publicationViewModel.Permissions.CanCreateReleases);
+                Assert.False(publicationViewModel.Permissions.CanAdoptMethodologies);
+                Assert.False(publicationViewModel.Permissions.CanCreateMethodologies);
+                Assert.False(publicationViewModel.Permissions.CanManageExternalMethodology);
+            }
+        }
+
+        [Fact]
+        public async Task ListPublications_CannotViewAllPublications()
+        {
+            var userId = Guid.NewGuid();
+
+            var topic = new Topic
+            {
+                Title = "Topic title",
+                Theme = new Theme
+                {
+                    Title = "Theme title",
+                },
+            };
+
+            var publication = new Publication
+            {
+                Title = "Test Publication",
+                Summary = "Test summary",
+                Slug = "test-slug",
+                Topic = topic,
+                Contact = new Contact
+                {
+                    ContactName = "contact name",
+                    ContactTelNo = "1234",
+                    TeamName = "team name",
+                    TeamEmail = "team@email",
+                },
+                SupersededBy = null,
+            };
+
+            var userService = new Mock<IUserService>(Strict);
+
+            userService.Setup(s => s.GetUserId()).Returns(userId);
+            userService.Setup(s => s.MatchesPolicy(CanAccessSystem)).ReturnsAsync(true);
+            userService.Setup(s => s.MatchesPolicy(CanViewAllPublications)).ReturnsAsync(false);
+
+            var contentDbContextId = Guid.NewGuid().ToString();
+            await using (var contentDbContext = InMemoryApplicationDbContext(contentDbContextId))
+            {
+                await contentDbContext.Publications.AddRangeAsync(publication);
```

Comment:
This isn't a very thorough test as we don't save any other publications that the user shouldn't be able to see.